### PR TITLE
Add message module for terminal output

### LINKED ISSUE #219: Standardize terminal messaging output function
Rather than calling `println!`, it would be nice to call something like `message.success` with some parameters that standardize the formatting of the output for e.g. `success`, `info`, `error`, etc.

### PR DESCRIPTION
**Pull request by ashleymichal**
fixes #219 

* include semantic commands for success, info, warn, etc.
* alphabetize emoji 🤓 
* replace some remaining inline emoji

### CODE REVIEW

**Comment by ashleygwilliams** [comment]
looks like you have an import error and some dead code warnings! lemme know if you need help with those :)

**Comment by ashleygwilliams** [comment]
adding @granjef3 as a reviewer as we'll tap him to finish out this migration :)

File: src/terminal/message.rs [hunk 1]
```diff
@@ -0,0 +1,34 @@
+use super::emoji;
+use console::Emoji;
+
+fn message(label: Emoji, msg: &str) {
+    println!("{}", format!("{0} {1} {0}", label, msg));
```
Comment (granjef3) [inline src/terminal/message.rs]:
```suggestion
    println!("{0} {1} {0}", label, msg);
```

File: src/commands/build/wranglerjs/bundle.rs [hunk 2]
```diff
@@ -254,7 +254,7 @@ mod tests {
     fn cleanup(name: String) {
         let current_dir = env::current_dir().unwrap();
         let path = Path::new(&current_dir).join(name);
-        println!("p: {:?}", path);
+        message::info(format!("p: {:?}", path));
```
Comment (granjef3) [inline src/commands/build/wranglerjs/bundle.rs]: need to `use crate::terminal::message` at the top of this file

File: src/terminal/emoji.rs [hunk 3]
```diff
@@ -13,16 +13,20 @@
...
 pub static DANCERS: Emoji = Emoji("👯 ", "");
+pub static EYES: Emoji = Emoji("👀 ", "");
+pub static FACEPALM: Emoji = Emoji("🤦‍♀️ ", "");
+pub static INBOX: Emoji = Emoji("📥 ", "");
+pub static INFO: Emoji = Emoji("💁‍ ", "");
 pub static MICROSCOPE: Emoji = Emoji("🔬 ", "");
-pub static CRAB: Emoji = Emoji("🦀 ", "");
+pub static SHEEP: Emoji = Emoji("🐑 ", "");
 pub static SLEUTH: Emoji = Emoji("🕵️‍♂️", "");
+pub static SNAIL: Emoji = Emoji("🐌 ", "");
```
Comment (granjef3) [inline src/terminal/emoji.rs]: Unused -- makes clippy complain.

File: src/terminal/emoji.rs [hunk 4]
```diff
@@ -13,16 +13,20 @@
 
 use console::Emoji;
 
-pub static WORKER: Emoji = Emoji("👷 ", "");
-pub static SPARKLES: Emoji = Emoji("✨  ", "");
+pub static CONSTRUCTION: Emoji = Emoji("🚧 ", "");
+pub static CRAB: Emoji = Emoji("🦀 ", "");
 pub static DANCERS: Emoji = Emoji("👯 ", "");
+pub static EYES: Emoji = Emoji("👀 ", "");
+pub static FACEPALM: Emoji = Emoji("🤦‍♀️ ", "");
```
Comment (granjef3) [inline src/terminal/emoji.rs]: Unused -- makes clippy complain.

File: src/terminal/message.rs [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+
+pub fn info(msg: &str) {
+    message(emoji::INFO, msg);
+}
+
+pub fn success(msg: &str) {
+    message(emoji::SPARKLES, msg);
+}
+
+pub fn warn(msg: &str) {
```
Comment (granjef3) [inline src/terminal/message.rs]: Unused -- makes clippy complain.

File: src/terminal/message.rs [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+    message(emoji::WARN, msg);
+}
+
+pub fn user_error(msg: &str) {
+    message(emoji::EYES, msg);
+}
+
+pub fn service_error(msg: &str) {
```
Comment (granjef3) [inline src/terminal/message.rs]: Unused -- makes clippy complain.

Comment (ashleymichal) [reply src/terminal/message.rs]: There isn't currently a place where these would be used with a println!, only with failure::bail. I agree keeping them would be ideal, i kinda want the equivalent of an eslintignore here though.

Comment (granjef3) [reply src/terminal/message.rs]: there is #[allow(dead_code)]

Comment (ashleygwilliams) [reply src/terminal/message.rs]: i do not think we should commit dead code. i think the pattern here is clear enough that someone can add. if there's a space in the code to use it, put it in that one spot.

it's very hard to keep track of and clean up the allow dead code attributes

**Pull request merged**